WD-Firmware/demo/build/demos/demo_comrv_baremetal.py

```diff
@@ -27,8 +32,18 @@ def __init__(self):
         'D_TICK_TIME_MS=4',
         'D_ISR_STACK_SIZE=400',
     ]
-      
+
     self.listSconscripts = [
       'comrv_baremetal',
-      'psp'
-    ]
\ No newline at end of file
+      'psp',
+      'demo_comrv_baremetal',
+    ]
+
+    self.listDemoSpecificCFlags = [
+      '-fcomrv',

```

Comment:
add comment to explain -fcomrv. this is user flag, we want them to understand

Reply:
done in caec55c